CITATION.cff

```diff
@@ -0,0 +1,22 @@
+cff-version: 1.2.0
+message: "If you want to cite IREE, feel free to use this"
+title: "IREE"
+abstract: >-
+  An MLIR-based compiler and runtime for ML models from multiple frameworks.
+date-released: 2019-09-18
+authors:
+  - name: "The IREE Team and contributors"
+  - family-names: "Vanik"
+    given-names: "Ben"
+    affiliation: Google
+  - family-names: "Laurenzo"
+    given-names: "Stella"
+    affiliation: Google
+license: "Apache-2.0"
```

Comment:
I think we should use the SPIDX identifier here

```suggestion
license: "Apache-2.0 WITH LLVM-exception"
```

This is what is suggested by the schema docs: https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#license